### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by pieterlange**
###  Summary
Makes it easier to run this software in an isolated environment.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/{project_slug}/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

> The following point can be removed after setting up CI (such as Travis) with coverage reports (such as Codecov)

* [x] I've written tests to cover the new code and functionality included in this PR.

> The following point can be removed after setting up a CLA reporting tool such as cla-assistant.io

* [x] I've read, agree to, and signed the [Contributor License Agreement (CLA)](https://cla-assistant.io/{project_slug}).

### CODE REVIEW

**Comment by pieterlange** [comment]
Please excuse the hack i used to dump the `env` into the dotenv file. If we can somehow make the app read from the real environment that'd be better but PHP is not really my native language :)

Where can i sign the CLA?

**Comment by pieterlange** [comment]
@fyunaz Can i do something to unblock this? ;) I assume you're busy.

**Comment by JoelPagliuca** [comment]
ran the project against this Dockerfile, and can confirm it works

**Comment by aoberoi** [comment]
@pieterlange are you able to visit the following URL to sign the CLA? https://cla-assistant.io/slackhq/goSDL?pullRequest=7

I think there might be an issue with the service we use, cla-assistant.io, for posting the status check on PRs.

**Comment by pieterlange** [comment]
CLA works now, thanks for the response @aoberoi

File: README.md [hunk 1]
```diff
@@ -122,6 +122,12 @@ Settings:
 
 6. Visit http://localhost:8000/sdl.php
 
+### Usage with docker
+1. Build locally: `docker build -t goSDL .`
```
Comment (fyunaz) [inline README.md]: When running this I got an error `invalid reference format: repository name must be lowercase`
Should we changes this to all lowercase? `docker build -t gosdl .`

File: README.md [hunk 2]
```diff
@@ -122,6 +122,12 @@ Settings:
...
+
+2. Run it: `docker run -ti --rm --env-file <your dotenv> -p 8080:8080 goSDL`
```
Comment (fyunaz) [inline README.md]: Same like above, getting an error of lowercase `goSDL`. Should we change this to?
`docker run -ti --rm --env-file <your dotenv> -p 8080:8080 gosdl`

File: README.md (hunk 1 above)
Comment (pieterlange) [reply README.md]: :+1:

**Pull request merged**